notebooks/kaggle/store-sales-time-series-forecasting/Time_Series_Forecasting.ipynb

```diff
@@ -0,0 +1,1515 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>As we can see from these characteristics, more than 25% of observations have 0 in 'sales' column. Maybe this can be influenced by some categoies, which mostly have zero number of sales. It is an interesting point to explore. <br>What is also interesting about 'sales' column is that 87.5% of observations have less than 640 number of sales, but <b>max</b> value is much bigger. Maybe it is connected with some very popular categories, maybe it is a consequence of some event (I can only guess now, but it can be connected with the earthquake in Ecuador, which happened in Ecuador in 2016. If it is true, such observations could be classified as outliers, because they won't help us with building a predictive model).</i>\n",
```

Comment:
I think it's worth adding a vertical line corresponding to the earthquake to time-series plots. This can be achieved with `ax.vline` function in matplotlib (`linestyle='--'` parameter might help to make it less noticeable).